include/ibamr/IBFEMethod.h

```diff
@@ -821,8 +821,21 @@ class IBFEMethod : public IBStrategy
     void putToDatabase(SAMRAI::tbox::Pointer<SAMRAI::tbox::Database> db) override;
 
     /*!
-     * Write the equation_systems data to a restart file in the specified directory.
-     */
+    * For technical reasons this class does not use SAMRAI's  RestartManager, so 
+    * restart files restart files must for the IBFE objects separately. The function 
+    * saves the solutions to the defined EquationSystems in an xdr file for each FE 
+    * part at each restart dump interval. An example snippet is included below to show 
```

Comment:
Does the restart dump interval matter to this function? The interval is used in the `if` statement, not inside the function itself.